src/rabbit_log.erl

```diff
@@ -73,14 +73,13 @@ log(Category, Level, Fmt, Args) when is_list(Args) ->
     end,
     lager:log(Sink, Level, self(), Fmt, Args).
 
-make_internal_sink_name(rabbit_log_connection) -> rabbit_log_connection_lager_event;
-make_internal_sink_name(rabbit_log_channel) -> rabbit_log_channel_lager_event;
-make_internal_sink_name(rabbit_log_mirroring) -> rabbit_log_mirroring_lager_event;
-make_internal_sink_name(rabbit_log_queue) -> rabbit_log_queue_lager_event;
-make_internal_sink_name(rabbit_log_federation) -> rabbit_log_federation_lager_event;
-make_internal_sink_name(rabbit_log_upgrade) -> rabbit_log_upgrade_lager_event;
-make_internal_sink_name(Category) ->
-    lager_util:make_internal_sink_name(Category).
+make_internal_sink_name(connection) -> rabbit_log_connection_lager_event;
+make_internal_sink_name(channel)    -> rabbit_log_channel_lager_event;
+make_internal_sink_name(mirroring)  -> rabbit_log_mirroring_lager_event;
+make_internal_sink_name(queue)      -> rabbit_log_queue_lager_event;
+make_internal_sink_name(federation) -> rabbit_log_federation_lager_event;
+make_internal_sink_name(upgrade)    -> rabbit_log_upgrade_lager_event;
+make_internal_sink_name(Category)   -> error({unknown_category, Category}).
```

Comment:
Why making this an error, instead of calling `lager_utils:make_internal_sink_name()`?

And currently, if the category is unknown, the log message is lost; here is what is logged in my case:
```
2017-08-08 09:17:28.174 [error] <0.72.0> FORMAT ERROR: {unknown_category,connection2} []
```